package.json

```
@@ -1,28 +1,29 @@
 {
-    "name": "connect-four-reboot",
-    "version": "0.0.1",
-    "description": "A CLI program for playing the game Connect Four.",
-    "main": "dist/index.js",
-    "type": "module",
-    "scripts": {
-        "build": "tsc",
-        "start": "node dist/index.js",
-        "dev": "node --loader ts-node/esm --no-warnings=ExperimentalWarning src/index.ts",
-        "test": "vitest run",
-        "test:watch": "vitest",
-        "format": "prettier --write \"src/**/*.{js,ts,jsx,tsx,json,css,scss,md}\""
-    },
-    "keywords": [
-        "typescript",
-        "cli"
-    ],
-    "author": "Marmelab",
-    "license": "MIT",
-    "devDependencies": {
-        "@types/node": "^20.4.5",
-        "prettier": "^3.4.2",
-        "ts-node": "^10.9.1",
-        "typescript": "^5.1.6",
-        "vitest": "^0.34.1"
-    }
+  "name": "connect-four-reboot",
+  "version": "0.0.1",
+  "description": "A CLI program for playing the game Connect Four.",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index.js",
+    "dev": "node --loader ts-node/esm --no-warnings=ExperimentalWarning src/index.ts",
```

```suggestion
    "dev": "node --inspect --loader ts-node/esm --no-warnings=ExperimentalWarning src/index.ts",
```

I think you should pass this additional argument to make the debugger works